modules/ts-capi-cli.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cluster_api_machine_management/cluster-api-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="ts-capi-cli_{context}"]
+= Referencing the intended objects when using the CLI
+
+For clusters that use the Cluster API, {oc-first} commands prioritize Cluster API objects over Machine API objects.
```

Comment:
Ahh, there it is! Wonder if this behavior can be linked somewhere in here: https://72885--ocpdocs-pr.netlify.app/openshift-enterprise/latest/machine_management/cluster_api_machine_management/cluster-api-about.html#cluster-api-overview_cluster-api-about

Reply:
Good call - not _exactly_ a limitation but a note up front about mixed CAPI/MAPI clusters could help avoid some pain